docs/_user_guide/best_practices_screen.md

```diff
@@ -2,102 +2,62 @@
 
 Generally speaking, you can use the `screen` command, described in more detail below, to continue having your scripts run on the Yens after logging out, but a very important first question to ask is:
```

Comment:
I'd love to have an intro about using slurm instead of doing this for most use cases